packages/rx_bloc_cli/mason_templates/bricks/feature_login/brick.yaml

```diff
@@ -0,0 +1,9 @@
+name: feature_login
+description: Login brick. Provides a way to log in the app with email and password.
```

Comment:
You can remove the period after `Login brick` and it would still make sense.

`Login brick provides a way to log in the app with email and password`